File: cookbook/java/Imports/getImports.java
package Imports;

import Utils.Request;

import java.io.FileInputStream;
import java.io.IOException;

import java.util.Map;
import java.util.Properties;

public class getImports
{
    public static void main(String[] args) throws Exception
    {
        Properties properties = new Properties();
        try {
            properties.load(new FileInputStream("config.properties"));
        } catch (IOException e) {
            System.out.print(e.getMessage());
        }
        String xKey = properties.getProperty("xKey");
        String baseUrl = properties.getProperty("url");

        //Lancement de la connexion pour remplir la requete
        String url = baseUrl + "imports/";

        Map<String, String> resp = Request.connection(url, xKey, null, "GET");

        if (resp.get("mess").equals("OK")) {
            //Lecture des donnees
            System.out.print(resp.get("reply") + "\n");
        }
        else {
            System.out.print("Error " + resp.get("code") + ": " + resp.get("mess") + "\n");
        }
    }
}

File: Csharp/Imports/getImports.cs
using System;
using System.Net;
using System.Xml;

namespace getImports
{
    internal class Program
    {
        private static void Main(string[] args)
        {
            // Changez le Path pour correspondre à la destination de votre fichier de configuration
            var doc = new XmlDocument();
            doc.Load("config.xml");

            var xKey = doc.DocumentElement.SelectSingleNode("/config/xKey").InnerText;
            var baseUrl = doc.DocumentElement.SelectSingleNode("/config/url").InnerText;

            // Lancement de la connexion pour remplir la requete
            var url = baseUrl + "imports/";

            var result = Utils.allConnection(url, xKey, null, "GET");

            //Verification des reponses
            if ((int)result[0] == 200)
            {
                Console.Write("{0}", (string)result[3]);
            }
            else
            {
                //Affichage de l'erreur
                Console.Write("Error : {0} {1}", (int)result[0], ((HttpWebResponse)result[2]).StatusCode.ToString());
            }

            Console.ReadLine();
            ((HttpWebResponse)result[2]).Close();
        }
    }
}
